content/enterprise_influxdb/v1.9/administration/manage/users-and-permissions/authorization-influxql.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Manage authorization with InfluxQL
+description: >
+  Manage users and permissions with InfluxQL.
+menu:
+  enterprise_influxdb_1_9:
+    parent: Manage users and permissions
+weight: 40
+related:
+  - /enterprise_influxdb/v1.9/administration/manage/security/authorization-api.md
+  - /{{< latest "chronograf" >}}/administration/managing-influxdb-users/
+  - /enterprise_influxdb/v1.9/administration/manage/security/fine-grained-authorization/
+---
+
+{{% enterprise-warning-authn-b4-authz %}}
+
+{{% note %}}
+We recommend using [Chronograf](/{{< latest "chronograf" >}}/administration/managing-influxdb-users/)
+and/or the [Enterprise meta API](/enterprise_influxdb/v1.9/administration/manage/security/authentication_and_authorization-api/)
+to manage InfluxDB Enterprise users and roles.
+{{% /note %}}
+
+{{% warn %}}
+We recommend operators *do not* mix and match InfluxQL
```

Comment:
Good point. Let's start the sentence "Outside of creating users [link to influxql command to create user], we recommend..."

Also instead of a warning, maybe this should be a note?